src/pages/AddRecord/AddRecord.tsx

```diff
@@ -54,26 +53,27 @@ export type IsInputFocusType = {
 
 export default function AddRecord() {
   const { CELEBRATION } = TEXT_DETAILS
-  const [checkAllFilled, setCheckAllFilled] = useState<CheckAllType>({
-    input: '',
-    textArea: '',
-  })
-  const [formDatas, setFormDatas] = useRecoilState(formDataAtom)
+  const ID = LocalStorage.get('postId') as string
+  const isModify = LocalStorage.get('modifyMode') === 'true'
+
+  const navigate = useNavigate()
+
+  const [isInputsHasValue, setIsInputsHasValue] =
+    useState<IsInputsHasValueType>({
+      input: false,
+      textArea: false,
+    })
+  const [formData, setFormData] = useRecoilState(formDataAtom)
   const { selectedCategory, selectedColor, selectedIcon }: FormDataType =
-    formDatas
+    formData
   const [files, setFiles] = useState<File[]>([])
-  const navigate = useNavigate()
   const [isClickBackButton, setIsBackButton] = useState(false)
   const [isLoadingWhileSubmit, setIsLoadingWhileSubmit] = useState(false)
   const [isInputFocus, setIsInputFocus] = useState(false)
   const [isMobile, setIsMobile] = useState<boolean>(false)
```

Comment:
타입 추론으로 boolean 을 적용 안해줘도 될 것 같아요
```suggestion
  const [isMobile, setIsMobile] = useState(false)
```

Reply:
넵 이런부분은 추론에 맡기도록 할게요 앞으로